docker/db-init/db-init.sql

```diff
@@ -0,0 +1,12 @@
+ALTER USER admin PASSWORD 'admin';
+
+CREATE DATABASE web_store;
+GRANT ALL ON DATABASE web_store TO admin;
+
+CREATE USER service_web_store WITH PASSWORD 'service_web_store';
+
+CREATE SCHEMA IF NOT EXISTS web_store AUTHORIZATION admin;
+GRANT ALL ON SCHEMA web_store TO admin;
+GRANT ALL ON SCHEMA web_store TO service_web_store;
+
+ALTER ROLE service_web_store SET search_path = web_store;
```

Comment:
Here's an init script that runs when postgres launches in the container. We have the root admin user, and a user specifically for the service. I create the schema so it's ready to go, and grant perms plus set the search path for our service user to this schema. Will hopefully make queries less cumbersome. Feel free to change DB name, schema name, user details as you see fit!